app/scripts/modules/amazon/src/securityGroup/clone/cloneSecurityGroup.controller.js

```diff
@@ -80,7 +80,8 @@ module.exports = angular
     vm.upsert = function() {
       // <account-select-field> only updates securityGroup.credentials, but Orca looks at account* before looking at credentials
       // Updating the rest of the attributes to send the correct (expected) account for all attributes
-      const { credentials } = $scope.securityGroup;
+      const { credentials, name } = $scope.securityGroup;
```

Comment:
`name` is unused and is causing a linter error

Reply:
thanks!